aiohttp/client.py

```diff
@@ -134,7 +134,7 @@
 
 @dataclasses.dataclass(frozen=True)
 class ClientTimeout:
-    total: Optional[float] = None
+    total: Optional[float] = 5 * 60  # 5 minute default timeout
```

Comment:
On second thought, is this the expected behaviour if a user were to do something like
`ClientTimeout(sock_read=600)`?

i.e. They set sock_read to 10 mins, but total is still set to 5 mins, so it achieves nothing.

I'm not sure now if having this default value makes things simpler or more complex...
@webknjaz Any thoughts?